src/test/resources/feature/search.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Search functionality
+ @searchFunctionality
```

Comment:
Please avoid to have a lot of tags, and use something more generic, like your name. This will ensure creating a lot of Runners or change tags everytime you need to run specific test

Reply:
Done